sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -329,31 +343,32 @@ public void UpsertCdsSpreadCurves(string scope, DateTimeOffset effectiveAt, stri
                         },
                         definition: instrument));
         }
-        
+
         /// <summary>
         /// This method add instruments to a portfolio (specified by its scope and code),
         /// by creating a transaction request with the instrument's LUID.
         /// </summary>
-        private void AddInstrumentsTransactionToPortfolio(
+        public void AddInstrumentsTransactionToPortfolio(
             List<string> luids,
-            string portfolioScope,
-            string portfolioCode,
+            string scope,
+            string portfolioID,
             DateTimeOffset effectiveAt)
         {
             // CREATE instrument transaction request
-            var transactionRequests = luids.Select(luid => 
-                    BuildTransactionRequest(luid, 1, 0.0m, "USD", effectiveAt, "StockIn"))
+            var transactionRequests = luids.Select(luid =>
+                    BuildTransactionRequest(luid, 1, 0.0m, "USD", effectiveAt, "Buy"))
```

Comment:
Changing the transaction code from StockIn to Buy will result in a corresponding cash movement for the transaction, was that intended?

Reply:
No - will revert.